### PR TITLE
Fix mxpy path variable

### PR DESCRIPTION
**Pull request by popenta**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/sdk-py/installing-mxpy.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ id: installing-mxpy
 title: Installing mxpy
 ---
 
-[comment]: # (mx-abstract)
+[comment]: # "mx-abstract"
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/sdk-py/installing-mxpy.md:6]: revert these changes - they are needed by the bot that splits the docs into granular sections

File: docs/sdk-and-tools/sdk-py/installing-mxpy.md [hunk 2]
```diff
@@ -41,9 +44,10 @@ wget -O mxpy-up.py https://raw.githubusercontent.com/multiversx/mx-sdk-py-cli/ma
 python3 mxpy-up.py
 ```
 
-This will create a light Python virtual environment (based on `venv`) in `~/multiversx-sdk/mxpy-venv `and also include `~/multiversx-sdk`in your **`$PATH`** variable (by editing the appropriate `.profile` file).
+This will create a light Python virtual environment (based on `venv`) in `~/multiversx-sdk/mxpy-venv `. Since `mxpy v6.0.0` we do not alter the **`$PATH`** variable by default anymore.
+You'll need to manually include `~/multiversx-sdk` in your **`$PATH`** variable (by editing the appropriate `.profile` file).
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/sdk-py/installing-mxpy.md:48]: an example would help

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: docs/sdk-and-tools/sdk-py/installing-mxpy.md (hunk 1 above)
Comment (popenta) [reply docs/sdk-and-tools/sdk-py/installing-mxpy.md:6]: Reverted

File: docs/sdk-and-tools/sdk-py/installing-mxpy.md (hunk 2 above)
Comment (popenta) [reply docs/sdk-and-tools/sdk-py/installing-mxpy.md]: Added an example.

**Review by bogdan-rosianu — APPROVED** [review]

File: docs/sdk-and-tools/sdk-py/installing-mxpy.md [hunk 3]
```diff
@@ -44,10 +44,13 @@ wget -O mxpy-up.py https://raw.githubusercontent.com/multiversx/mx-sdk-py-cli/ma
 python3 mxpy-up.py
 ```
 
-This will create a light Python virtual environment (based on `venv`) in `~/multiversx-sdk/mxpy-venv `. Since `mxpy v6.0.0` we do not alter the **`$PATH`** variable by default anymore.
-You'll need to manually include `~/multiversx-sdk` in your **`$PATH`** variable (by editing the appropriate `.profile` file).
+This will create a light Python virtual environment (based on `venv`) in `~/multiversx-sdk/mxpy-venv `. Since `mxpy v6.0.0` we do not alter the **`$PATH`** variable.
+You'll need to manually include `~/multiversx-sdk` in your **`$PATH`** variable (by editing the appropriate `.profile` file) as done bellow:
+```
+export PATH="${{HOME}}/multiversx-sdk:${{PATH}}"
```
[the remark this answers is not in the document]
Comment (popenta) [reply docs/sdk-and-tools/sdk-py/installing-mxpy.md]: Indeed, fixed.

**Review by andreibancioiu — APPROVED** [review]
